Makefile

```diff
@@ -0,0 +1,17 @@
+export DEFAULT_HELP_TARGET ?= help
+export HELP_FILTER ?= help|init|codefresh/sync
+export DOCKER_NETWORK ?= default
+export BUILD_HARNESS_BRANCH ?= 0.15.3
```

Comment:
why do we need `BUILD_HARNESS_BRANCH`?

Reply:
this is how we pin build_harness version and here we need at least this version